components/button/button.css

```diff
@@ -0,0 +1,130 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+:root{
+    --color-primary : #FF3B30;
+    --color-secondary: grey;
+    --color-success: green;
+    --color-danger: red;
+    --color-warning: rgb(218, 218, 54);
+
+    --color-white: #Fff6ea;
+    --color-black: #282828;
+}
+
+button {
+    background-color: transparent;
+    border: none;
+    cursor: pointer;
+    outline: none;
+}
+
+.button-container{
+    margin: 1rem;
+}
+
+.btn {
+    background-color: none;
+    color: var(--color-white);
+    padding: 0.5rem 1rem;
+
+}
+.btn-large{
+    padding: 1rem 4rem;
+}
+.btn:hover{
+    background-color: var(--color-black);
+    /* opacity: 0.7; */
+}
```

Comment:
do you want all the buttons with btn class to have background color as black on hover? if yes then it's fine otherwise please check

Reply:
Yes, I found this to be easier. I overwrite the hover property on buttons which I want to behave differently and those are very few